app/frontend/src/components/autocomplete/autocomplete.js

```diff
@@ -1,37 +1,46 @@
-import 'es6-promise/auto';
-import view from './autocomplete.view';
+import accessibleAutocomplete from 'accessible-autocomplete';
+import 'accessible-autocomplete/dist/accessible-autocomplete.min.css';
+import './autocomplete.scss';
 
-export const renderAutocomplete = (widgetParams) => {
-  view.create(widgetParams.container, widgetParams.input, widgetParams.key);
+const AUTOCOMPLETE_THRESHOLD = 3;
```

Comment:
Not immediately obvious what this number represents. Can we rename it to something a bit clearer?